### ISSUE TITLE
feature request: 3d character rigging

### ISSUE THREAD

**Issue by cro3d** [MrForExample/ComfyUI-3D-Pack#96]
im not up to date on ML rigging, i remember https://zhan-xu.github.io/rig-net/ from 2020, but that would be useful.  
i guess control -net will pose a character but exporting a rigged character would be really cool.

**Comment by MrForExample** [comment]
Yeah, it would be, but unfortunately I have limited time & energy, so currently the best I can do is focus on 3D Gen directly related components.
However, feel free to add it as custom node if you want & can, cheers 👍
